Test.txt

```diff
@@ -1 +1,3 @@
-This is a test.
\ No newline at end of file
+This is how you change a file.
+This is the second line.
```

Comment:
Done.

---
In reply to: [59789925](https://github.com/kivancmuslu/CFTest/pull/1#discussion_r59789925) [](ancestors = 59789925)